.github/workflows/ci.yml

```diff
@@ -160,37 +160,16 @@ jobs:
 
       - run: docker buildx create --driver docker-container --use
 
-      - name: Build snuba docker image for CI
-        run: |
-          set -euxo pipefail
-
-          args=()
-
-          if [ ${{ github.event_name }} = 'push' ]; then
-            args+=(
-              --tag ghcr.io/getsentry/snuba-ci:latest
-              --cache-to type=registry,ref=ghcr.io/getsentry/snuba-ci:latest,mode=max
```

Comment:
This composite action uses gha cache: https://docs.docker.com/build/cache/backends/gha/